server/src/utils/account.js

```diff
@@ -32,17 +32,6 @@ const unlockAccount = async (accountId) => {
   console.log(`no account found to unlock with id ${accountId}`)
 }
 
-const unlockAccountConditionally = async (accountAddress) => {
```

Comment:
@LiorAgnin this was a fix we made to solve the issue with accounts getting locked forever and then admin jobs getting stuck in the queue. I'm afraid if we remove this we'll start to have the same issues as before don't you think?

Reply:
My bad. You're right. Will fix it.